docs/installation/server-config/security.rst

```diff
@@ -1,4 +1,242 @@
 .. _ksql-security:
 
 Securing KSQL
--------------
\ No newline at end of file
+=============
+
+KSQL supports many of the security features of both Apache Kafka and the |sr|.
```

Comment:
Is this `|sr|.` link meant to work in the staging docs?  Cos it doesn't in my browser :D

Reply:
it will work once all merged